src/shared/sam/debugger/awsSamDebugConfiguration.ts

```diff
@@ -40,6 +41,41 @@ export function isCodeTargetProperties(props: TargetProperties): props is CodeTa
     return props.target === CODE_TARGET_TYPE
 }
 
+/**
+ * Ensures that the `projectRoot` or `samTemplatePath` relative properties on
+ * the given `config` are relative (not absolute) paths.
+ *
+ * @param folder  Workspace folder, or empty to use the workspace associated with `projectRoot` or `samTemplatePath`.
+ * @param config
+ */
+export function ensureRelativePaths(
+    folder: vscode.WorkspaceFolder | undefined,
+    config: AwsSamDebuggerConfiguration
+): void {
+    if (config.invokeTarget.target !== 'template' && config.invokeTarget.target !== 'code') {
```

Comment:
Do we have constants for template and code ?  
Comment applies to whole method

Reply:
we do, though I keep forgetting about them because it doesn't really serve any purpose since `target` is type-checked.